secretmanager/Gemfile

```diff
@@ -14,7 +14,7 @@
 
 source "https://rubygems.org"
 
-gem "google-cloud-secret_manager", "~> 0.2"
+gem "google-cloud-secret_manager", "~> 0.3"
```

Comment:
```suggestion
gem "google-cloud-secret_manager"
```

Reply:
While we _are_ going to move the `google-cloud-secret_manager` gem to 1.0 shortly, I'd still rather keep the version restriction (and update it to `~> 1.0` at the appropriate time) as a best practice.

Reply:
Can it be merged then?

Reply:
The instruction recommend `bundle install` the ci uses `bundle update`, it will cause no issue to users to remove the version restriction in the Gemfile, and will alert us early if the sample stops working with the latest version

Reply:
Done

Reply:
@TheRoyalTnetennba Okay sounds good to me. As long as we don't recommend that _end users_ specify the dependency without a version restriction.